.github/workflows/publish.yml

```diff
@@ -20,7 +20,8 @@ jobs:
     with:
       package-name: http-sink
       branch: ${{ github.event.inputs.branch }}
-      fail-fast: false  
+      fail-fast: false
+      readme: '../../README.md'
```

Comment:
Is this path relative to the `Connector.toml`? or relative to where the command is running?

Reply:
To command is running! Yeah I think it should be `readme: './README.md'` on first sight I assumed the action to publish connectors changed directory to package's dir (being `crates/http-sink`) so I did `../../` but actually is not required. Fixing it..